dev/com.ibm.ws.ssl/resources/com/ibm/ws/ssl/resources/ssl.nlsprops

```diff
@@ -600,4 +600,11 @@ ssl.dynamicSelection.conflict.CWPKI0815W.useraction=Review the conflicting Outbo
 ssl.defaultOutbound.conflict.CWPKI0816W=CWPKI0816W: The outboundConnection element with an asterisk (*) as the host and port is set to the [{0}] SSL configuration. This setting is in conflict with the outboundSSLRef attribute setting. The {1} SSL configuration that is specified by the outboundSSLRef attribute is used.
 ssl.defaultOutbound.conflict.CWPKI0816W.explanation=The outboundConnection element with an asterisk (*) as the host and port is in conflict with the outboundSSLRef attribute that is configured. The server uses the SSL configuration that is specified by the outboundSSLRef attribute.
 ssl.defaultOutbound.conflict.CWPKI0816W.useraction=Review the conflicting configuration and determine which configuration to use as the default SSL configuration. Remove the configuration that you do not need.
-
+#--------------------------------------------------------------------------------------------------
+ssl.defaultKeyStore.expected.CWPKI0817W=CWPKI0817W: The default SSL configuration expects a <keyStore> element with an ID of {0} and a password. The {0} <keyStore> element is missing or the password is not specified. Either define the missing element: <keyStore id="{0}" password="..." /> or change the default SSL configuration to use an existing keystore: <ssl id="defaultSSLConfig" keyStoreRef="newKeyStore" />.
```

Comment:
Recommend a few changes in **bold:** (added a couple of commas)
The default SSL configuration expects a <keyStore> element **__with an id value of_**_ {0}, and a password. The {0} <keyStore> element is missing, or the password is not specified. Either define the missing element: <keyStore id="{0}" password="**_your_password_"** />, or change the default SSL configuration to use an existing keystore: <ssl id="defaultSSLConfig" keyStoreRef="newKeyStore"/>.

Reply:
the code snippet is not displaying in the comments--we want to be sure the <> will build from git.